aurelia-slickgrid/src/aurelia-slickgrid/sorters/sorterUtilities.ts

```diff
@@ -27,3 +28,19 @@ export function sortByFieldType(value1: any, value2: any, fieldType: FieldType,
 
   return sortResult;
 }
+
+export function compareDates(sortDirection, value1, value2, format, strict?: boolean) {
+  let diff = 0;
+
+  if (!moment(value1, format, strict).isValid()) {
```

Comment:
add the `null` check in both `if` condition, this will improve a lot performance. Please try with 5000+ rows and you should see a drastic improvement with the `null` check